src/urh/dev/native/lib/cbladerf.pxd

```diff
@@ -125,11 +126,11 @@ cdef extern from "libbladeRF.h":
     int bladerf_sync_rx(bladerf *dev, void *samples, unsigned int num_samples, bladerf_metadata *metadata, unsigned int timeout_ms)
     int bladerf_sync_tx(bladerf *dev, const void *samples, unsigned int num_samples, bladerf_metadata *metadata, unsigned int timeout_ms)
 
-IF BLADERF_API_VERSION >= 2:
-    ctypedef uint64_t bladerf_frequency
+IF BLADERF_API_VERSION >= 1.91:
```

Comment:
Can we set this back to the previous state?

Reply:
If I set it back to `IF BLADERF_API_VERSION >= 2`

Then when I compile for it BLADERF_API_VERSION >= 1.91` it throws the following message:

`Cannot assign type 'bladerf_frequency *' to 'uint64_t *'`